_posts/2017-02-02-using-discover-agrc-utah-gov-services-in-the-export-web-map-print-service.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+status: publish
+published: true
+title: 'Using discover.agrc.utah.gov Services PrintingTools/Export Web Map Service'
```

Comment:
I think we should shorten this title to something like 'Printing Web Maps with Discover Services'